sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -294,13 +293,13 @@ export class MessageSender extends LinkEntity<AwaitableSender> {
    */
   public async open(
     options?: AwaitableSenderOptions,
-    abortSignal?: AbortSignalLike
+    operationOptions?: OperationOptionsBase
```

Comment:
It's a bug if we don't pass these options, isn't it? Ie, are they actually optional?

Reply:
Made them non-optional.